src/lib/containers/UserCardList.tsx

```diff
@@ -150,7 +153,8 @@ export default class UserCardList extends React.Component<
                   disableLink={true}
                   hideEmail={true}
                   size={size}
-                  userProfile={fauxUserProfile}
+                  // TODO: Modify UserCard to accept a fake profile with no ownerId field, and remove this cast.
+                  userProfile={fauxUserProfile as UserProfile}
```

Comment:
UserCard works fine if this is null, but I think making that change type-safe is probably complicated. Not doing it now for the sake of keeping the scope of this PR small.